src/services/FirebaseService.ts

```diff
@@ -9,6 +8,8 @@ import { getFunctions, connectFunctionsEmulator } from "firebase/functions";
 import { Utils } from "services/utils";
 import { EventAggregator } from "aurelia-event-aggregator";
 
+type Address = string;
```

Comment:
I would like to have `Address` be defined somewhere else than `EthereumService`.
Reason: I cannot import `FirebaseService` easily into Cypress, because it pulls in `EthService`, so quickest solution was to remove the dep on `EthService`.

In general, reducing the import footprint of each file might be a good idea overall